src/dbus_api/macros.rs

```diff
@@ -84,3 +84,14 @@ macro_rules! properties_footer {
         }
     };
 }
+
+macro_rules! pool_op_logging {
+    ($pre_oper:tt $(, $pre_args:expr)*; $post_oper:tt $(, $post_args:expr)*; $engine_op:expr) => {{
+        info!($pre_oper, $($pre_args),*);
+        let result = $engine_op;
+        if let Ok(ref uuid) = result {
```

Comment:
```uuid```  isn't a good name for what is captured. It's actually an action, like "CreateAction", "DestroyAction", etc. Better to call it ```action```.

Reply:
Done.